src/05-derivatives/05-diffusion-derivatives-tractography.md

```diff
@@ -0,0 +1,137 @@
+-   [.trk - Diffusion Toolkit (+TrackVis)](https://web.archive.org/web/20190103230122/http://www.trackvis.org/docs/?subsect=fileformat)
+-   [.tck - MRtrix](https://web.archive.org/web/20190103230209/https://mrtrix.readthedocs.io/en/latest/getting_started/image_data.html#tracks-file-format-tck)
+
+### JSON Metadata
+
+We have two main classes of tractography algorithms: Global or Local
+tractography, each with various supported algorithms.
+
+| Class  | Algorithms | Reference                                                                                                                            |
+| ------ | ---------- | ------------------------------------------------------------------------------------------------------------------------------------ |
+| LOCAL  | PROB       | Sherbondy, A.J., et al. (2008); Tournier J.D. et al. (2012); Behrens et al. (2007)                                                   |
+| LOCAL  | DET        | Conturo, T. et al. (1999); Mori, S. et al. (1999)                                                                                    |
+| LOCAL  | EUDX       | Garyfallidis et al. (2010); Mori, S. et al. (1999)                                                                                   |
+| LOCAL  | FACT       | Jiang, H. et al. (2005)                                                                                                              |
+| LOCAL  | STT        | Lazar, M. et al. (2003)                                                                                                              |
+| LOCAL  | NULL       | Morris et al. (2008)                                                                                                                 |
+| GLOBAL | UKF        | Malcolm J-G. et al. (2009)                                                                                                           |
+| GLOBAL | SpinGlass  | Fillard P. et al. (2009)                                                                                                             |
+| GLOBAL | ENS        | Takemura et al. (2016)                                                                                                               |
+| GLOBAL | Other      | Mangin, J.F. et al. (2013); Aganj, I. et al. (2011); Neher, P.F. et al. (2012); Jbabdi, S., et al. (2007); Reisert, M. et al. (2011) |
+
+The JSON sidecar contains the following key/value pairs:
+
+| **Key name**       | **Description**                                                                                                      |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------- |
+| TractographyClass  | REQUIRED. Allowed values: `local`, `global`                                                                          |
+| TractographyMethod | REQUIRED. Allowed values: `probabilistic`,`deterministic`,`eudx`,`fact`,`stt`,`null`,`ukf`,`spinglass`,`ens`,`other` |
```

Comment:
We need to add "pft" (particle filtering tractography, see [Girard et al., 2014](https://www.ncbi.nlm.nih.gov/pubmed/24816531))